source/initial_temperature/S40RTS_perturbation.cc

```diff
@@ -380,6 +403,11 @@ namespace aspect
                              Patterns::Double (0),
                              "The value of the thermal expansion coefficient $\\beta$. "
                              "Units: $1/K$.");
+          prm.declare_entry ("Use thermal expansion coefficient from material model", "false",
+                             Patterns::Bool (),
+                             "Option to use the material model to determine the thermal expansion "
+                             "coefficient, instead of the thermal expansion coefficient specified in "
+                             "this model.");
```

Comment:
Maybe this is a bit clearer: "Option to take the thermal expansion coefficient from the material model instead of from what is specified in this section."